staff/lluis-bagur/pre-course/1-calculadora/calculadora.js

```diff
@@ -0,0 +1,86 @@
+calculadora.js
+
+var num1 = 1.2;
+var num2 = 3;
+
+// Son numeros?
+ var check1 = "";
+ var check2 = "";
+
+check1 = isNaN (num1);
+check2 = isNaN (num2);
+
+
+
+console.log("Los valores introducidos son: "+num1+" y "+num2);
+
+// hemos comprobado que los dos son numeros (afrimando que no son NaN)
+
+if (check1 === false && check2 === false) {

```

Comment:
Aquí no hacía falta poner 3 "=", con 2 funciona igual
También otra manera de escribirlo es:
if (!check1 && !check2){

Reply:
tree equals is better in this case, he's corresct

Reply:
doing that type of logic operations with a boolean is a bad practice, a better option is the one above me

Reply:
Poner 2 = no hace que el codigo vaya mas rapido y de hecho es siempre mejor poner 3 por si puede haber algun caso que los 2 = den un valor true cuando realmente es false.

"Javascsript had some design flaws and is quirky. This means that the type juggling that occurs with 2 equal signs is not always intuitive. We run the risk of things being evaluated as true when we would intuitively expect them to be false.

When designing important programmes you don't want to be playing Russian Roulette with your logic. So, its better for your sanity to get into the habit of writing strict code (with 3 equal signs)."